doc/src/devhost.rst

```diff
@@ -112,10 +112,10 @@ A development environment (e.g. named ``dev``) would typically look like this:
    method = fc-nixos-dev-vm
    # The release has the format fcrs://channel/release
    # release can be empty, than the latest release for the channel is used.
-   release = fcrs://fc-23.11-production/2024_001
+   release = fcrs://fc-24.05-production/2024_021
    host = dev.example.com
 
-   channel = https://hydra.flyingcircus.io/build/116563/download/1/nixexprs.tar.xz
+   channel = https://hydra.flyingcircus.io/build/457353/download/1/nixexprs.tar.xz
```

Comment:
I am not fully certain about this here: What is the role of the `channel` specification now that we already have a `release = fcrs://fc-24.05-production/2024_021` that points to metadate holding the same channel information?